software/control/core/core.py

```diff
@@ -4623,79 +4619,66 @@ def __init__(
         self.liveController = liveController
         self.stage = stage
         self.piezo = piezo
+        self.objectiveStore = objectiveStore
+        self.laserAFSettingManager = laserAFSettingManager
 
         self.is_initialized = False
-        self.x_reference = 0
-        self.pixel_to_um = 1
-        self.x_offset = 0
-        self.y_offset = 0
+
+        self.laser_af_properties = LaserAFConfig()
         self.x_width = 3088
         self.y_width = 2064
 
         self.spot_spacing_pixels = None  # spacing between the spots from the two interfaces (unit: pixel)
 
-        self.look_for_cache = look_for_cache
-
         self.image = None  # for saving the focus camera image for debugging when centroid cannot be found
 
-        if look_for_cache:
-            cache_path = "cache/laser_af_reference_plane.txt"
-            try:
-                with open(cache_path, "r") as cache_file:
-                    for line in cache_file:
-                        value_list = line.split(",")
-                        x_offset = float(value_list[0])
-                        y_offset = float(value_list[1])
-                        width = int(value_list[2])
-                        height = int(value_list[3])
-                        pixel_to_um = float(value_list[4])
-                        x_reference = float(value_list[5])
-                        self.initialize_manual(x_offset, y_offset, width, height, pixel_to_um, x_reference)
-                        break
-            except (FileNotFoundError, ValueError, IndexError) as e:
-                print("Unable to read laser AF state cache, exception below:")
-                print(e)
-                pass
+        # Load configurations if provided
+        if self.laserAFSettingManager:
+            self.laser_af_settings = self.laserAFSettingManager.get_laser_af_settings()
+            self.load_cached_configuration()
```

Comment:
`self.laser_af_settings = self.laserAFSettingManager.get_laser_af_settings()` should be absorbed into `self.load_cached_configuration()`. Also `self.laser_af_settings` should be replaced with `laser_af_settings`